stat/distuv/general.go

```diff
@@ -20,5 +20,7 @@ var (
 )
 
 var (
-	expNegOneHalf = math.Exp(-0.5)
+	expNegOneHalf   = math.Exp(-0.5)
+	eulerMascheroni = 0.5772156649015328606065120900824024310421
```

Comment:
These can be consts. Add `// https://oeis.org/A001620`?

Reply:
Done.